frontend/src/routes/Video.tsx

```diff
@@ -252,11 +252,33 @@ const VideoPage: React.FC<Props> = ({ eventRef, realmRef, basePath }) => {
     const breadcrumbs = (realm.isRoot ? realm.ancestors : realm.ancestors.concat(realm))
         .map(({ name, path }) => ({ label: name, link: path }));
 
-
+    const { hasStarted, hasEnded } = getEventTimeInfo(event);
+    const isCurrentlyLive = hasStarted === true && hasEnded === false;
+
+    const structuredData = {
+        "@context": "https://schema.org",
+        "@type": "VideoObject",
+        name: event.title,
+        description: event.description,
+        thumbnailUrl: event.syncedData.thumbnail,
+        uploadDate: event.created,
+        duration: toIsoDuration(event.syncedData.duration),
+        ...event.isLive && event.syncedData.startTime && event.syncedData.endTime && {
+            publication: {
+                "@type": "BroadcastEvent",
+                isLiveBroadcast: isCurrentlyLive,
+                startDate: event.syncedData.startTime,
+                endDate: event.syncedData.endTime,
+            },
+        },
+        // TODO: Provide `contentUrl` or `embedUrl`? Google docs say one should,
+        // but it's not clear what for.
```

Comment:
They probably want to be able to show/embed it. :shrug: Or maybe even process it. If you hover over videos in Google's video search, for example, you see the familiar slideshow-like preview many video pages use.

Once #557 lands, we can just put this as `embedURL`, right? :thinking: I mean why not. :shrug:

Reply:
Yep, thats roughly what they do with the contentUrl. I think we should not do that generally as I think some universities would dislike their video portal "voluntarily giving the video to Google". Of course they can't really avoid that anyway and it's not really useful to say that. Buut you know.

`embedUrl` after #557 sounds good.

Reply:
:+1: